src/include/mujoco.cmake

```diff
@@ -1,11 +1,17 @@
+if(CMAKE_SYSTEM_NAME MATCHES "Linux")
+  set(MUJOCO_PATH /opt/mujoco-3.2.6-linux-x86_64/mujoco-3.2.6/)
+elseif (CMAKE_SYSTEM_NAME MATCHES "Darwin")
+  set(MUJOCO_PATH /Application/MuJoCo.app/mujoco.framework)
```

Comment:
According to [Mujoco docs](https://mujoco.readthedocs.io/en/stable/programming/index.html#getting-started) this is should be the path to the mujoco shared library after installing it to Application. Can you verify?